core/src/test/java/org/apache/calcite/util/Smalls.java

```diff
@@ -182,6 +184,10 @@ public void close() {
     };
   }
 
+  public static QueryableTable generateStrings2(final List<Integer> list) {
+    return generateStrings(list.size());
```

Comment:
Can we give it a more meaning name ? not XXX2 ?

Reply:
good catch, done.